### PR TITLE
fix default compose up command

### PR DESCRIPTION
**Pull request by ravipal**
Fixes: #1954

### CODE REVIEW

**Comment by bwateratmsft** [comment]
I'd be OK with taking this for 1.2.0.

File: src/commands/selectCommandTemplate.ts [hunk 1]
```diff
@@ -26,8 +26,8 @@ const defaults: { [key in TemplateCommand]: CommandTemplate } = {
     'runInteractive': { label: 'Docker Run (Interactive)', template: 'docker run --rm -it ${exposedPorts} ${tag}' },
     'attach': { label: 'Docker Attach', template: 'docker exec -it ${containerId} ${shellCommand}' },
     'logs': { label: 'Docker Logs', template: 'docker logs -f ${containerId}' },
-    'composeUp': { label: 'Compose Up', template: 'docker-compose up ${configurationFile} ${detached} ${build}' },
-    'composeDown': { label: 'Compose Down', template: 'docker-compose down ${configurationFile}' },
+    'composeUp': { label: 'Compose Up', template: 'docker-compose ${configurationFile} up ${detached} ${build}' },
```
Comment (bwateratmsft) [inline src/commands/selectCommandTemplate.ts:29]: These are already corrected in `package.json` right?

Comment (bwateratmsft) [reply src/commands/selectCommandTemplate.ts:29]: Yep, they are.

**Comment by ravipal** [comment]
> I'd be OK with taking this for 1.2.0. Anyone else agree/disagree?

This is really a corner case. I am fine either way.

**Pull request merged**